src/components/Home.js

```diff
@@ -18,10 +40,11 @@ const Home = () => {
             crime statistics.
           </p>
           <img
-            className='absolute right-0'
+            className='absolute right-0 cursor-pointer'
             style={{ bottom: '-30px' }}
             src='/img/arrow-down-circle.svg'
             alt='more information below'
+            onClick={scrollToBottom}
```

Comment:
I tried this, but I don't think it will work since we are using the hash router and can't have two `#` fragments